packages/mdc-slider/_mixins.scss

```diff
@@ -48,3 +48,15 @@
   }
 }
 
+@mixin mdc-slider-value-label-color($color) {
```

Comment:
The default should be that changing the value label color also updates the thumb to the same color, so `@include` the `mdc-slider-thumb-color` mixin inside this one

Reply:
Done!